app/assets/stylesheets/custom.css.scss

```diff
@@ -133,35 +139,40 @@ body {
   padding: 0;
   font-size: 4.5em;
   font-weight: 700;
-  color: gray;
+  color: $muted;
 }
 
 /* Custom Button Styles */
 
-.btn-dark {
-  border-radius: 0;
-  color: $white;
-  background-color: rgba(0,0,0,0.4);
+.btn-xlarge {
+  padding: 18px 28px;
+  font-size: 22px;
+  line-height: normal;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  border-radius: 8px;
 }
 
-.btn-dark:hover,
-.btn-dark:focus,
-.btn-dark:active {
-  color: $white;
-  background-color: rgba(0,0,0,0.7);
+.btn-ghost {
+  color: $primaryColor !important;
+  background-color: rgba(0,0,0,0) !important;
+  border: 1px solid $primaryColor !important;
+  font-weight: bold;
+  border-radius: 0 !important;
 }
 
-.btn-light {
-  border-radius: 0;
-  color: #333;
-  background-color: rgb(255,255,255);
+.btn-ghost:hover,
+.btn-ghost:focus,
+.btn-ghost:active {
+  color: $hoverColor !important;
```

Comment:
!important should not be used<br>Name of variable `hoverColor` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered border, color, text-decoration